tests/integration-tests/big-decimal/test/test.js

```diff
@@ -0,0 +1,77 @@
+const path = require("path");
```

Comment:
I really like that we moved to integration tests, and that I consistently see new integration tests being added with your changes.

The amount of code that these require as compared to the size of the change is intimidating. Is there a way to move a lot of this into a library or something?

Reply:
I carn try to move the contract, migration, and other files that repeat between many tests to a shared directory. The `waitForSubgraphToBeSynced` function can also be shared.